packages/contracts/contracts/ZUSDToken.sol

```diff
@@ -262,12 +258,6 @@ contract ZUSDToken is ZUSDTokenStorage, CheckContract, IZUSDToken, Ownable {
             _recipient != address(0) && _recipient != address(this),
             "ZUSD: Cannot transfer tokens directly to the ZUSD token contract or the zero address"
         );
-        require(
```

Comment:
Why this validation was removed?

Reply:
because it consumes a lot of gas on each call checking a pretty unlikely condition to happen 
it was agreed and reflected in the PRD.